packages/spor-expandable-react-native/src/ExpandableItem.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Box } from "@vygruppen/spor-layout-react-native";
+import { Text } from "@vygruppen/spor-typography-react-native";
+import { spacing } from "@vygruppen/spor-theme-react-native/src/foundations/spacing";
+
+
+type ExpandableItemProps = {
+    children: React.ReactNode,
+};
+export const ExpandableItem = ({
+    children
+}: ExpandableItemProps) => {
+    return (< Box style={{ marginBottom: spacing.sm, marginLeft: spacing.sm }} >
+        <Text>{children}</Text>
```

Comment:
Jeg ønsker egentlig å ikke ha <Text> her. Noen som har tips til hvordan jeg kan skrive slik at man kan sende in forskjellige komponenter ? 😀

Reply:
Skal man kunne sende inn bare en string her?

Reply:
Jeg ser for meg at brukeren selv kan sende inn `<Text>` rundt komponentene sine her, så de kan bestemme om det f.eks. skal være flere avsnitt med tekst.